os-boot/opensbi/sail.dts

```diff
@@ -14,10 +17,16 @@
       reg = <0>;
       status = "okay";
       compatible = "riscv";
-      riscv,isa = "rv64imac";
+      riscv,isa-base = "rv64i";
```

Comment:
This can't boot FreeBSD, it currently only recognises riscv,isa.

Reply:
Also, why is the FDT within opensbi/?

Reply:
> This can't boot FreeBSD, it currently only recognises riscv,isa.

I just test openSBI here, let FreeBSD to another pr(